.github/workflows/on_pull_request.yml

```diff
@@ -55,6 +55,21 @@ jobs:
               - '**/CMakeLists.txt'
               - '**/*cmake'
 
+  python-paths-filter:
+    runs-on: ubuntu-latest
+    outputs:
+      python_changes: ${{ steps.filter.outputs.python_changes }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.ref || '' }}
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            python_changes:
+              - '**/*.py'
```

Comment:
Should probably also trigger if a `requirements.txt` or `pyproject.toml` file changes

Reply:
👍